src/i18n/en.ts

```diff
@@ -223,6 +223,11 @@ const en = {
       "Clears the queue of processes that are waiting to be executed in the background",
     "administration.maintenance.clearLongRunningTasksQueue.success":
       "Background process queue successfully cleared.",
+    "administration.maintenance.reloadFTS": "Reload Full Text Search",
+    "administration.maintenance.reloadFTS.tooltip":
+      "Triggers a new initialization of Lucene indexes if they are used.",
```

Comment:
I suggest "Triggers re-initialization of full-text search indexes, if available."